WalletWasabi.Gui/Controls/WalletExplorer/TransactionBroadcasterViewModel.cs

```diff
@@ -82,8 +82,7 @@ public TransactionBroadcasterViewModel() : base("Transaction Broadcaster")
 
 			BroadcastTransactionCommand = ReactiveCommand.CreateFromTask(
 				async () => await OnDoTransactionBroadcastAsync(),
-				this.WhenAny(x => x.TransactionString, (transactionString) => (!string.IsNullOrWhiteSpace(transactionString.Value)))
-					.ObserveOn(RxApp.MainThreadScheduler));
+				this.WhenAny(x => x.TransactionString, (transactionString) => !string.IsNullOrWhiteSpace(transactionString.Value)).ObserveOn(RxApp.MainThreadScheduler));
```

Comment:
This is a must-have?

Reply:
I just tried. Yes, because it's a parameter, so it's confusing to do multiple lines.

![image](https://user-images.githubusercontent.com/9156103/73070584-0c887d00-3e87-11ea-9041-e45bc5fe871b.png)

Your issue is that it's too long. For that I it should be better to take it out to a variable. I'm doing it.

Reply:
Done.
![image](https://user-images.githubusercontent.com/9156103/73070733-7d2f9980-3e87-11ea-94e5-f3c8ce627047.png)